README.md

```diff
@@ -23,6 +23,7 @@ To get started, see the following resources:
 * [Amazon Chime SDK Developer Guide](https://docs.aws.amazon.com/chime/latest/dg/meetings-sdk.html)
 * [Amazon Chime SDK API Reference](http://docs.aws.amazon.com/chime/latest/APIReference/Welcome.html)
 * [Amazon Chime SDK for JavaScript Documentation](https://aws.github.io/amazon-chime-sdk-js)
+* [Amazon Chime SDK for JavaScript Supported browser](https://docs.aws.amazon.com/chime/latest/dg/meetings-sdk.html#mtg-browsers)
```

Comment:
`Supported Browsers` and could we move this up to between `Pricing` and `Developer Guide`?

Reply:
Updated.